### PR TITLE
Add Arabic text to WordFont#as_json

### PR DESCRIPTION
**Pull request by maryamklabib**
This is a short fix to show the Arabic text as requested in https://github.com/quran/quran.com-api/issues/44#issuecomment-229146160.

Can consider merging in the Token class as a separate commit.

### CODE REVIEW

File: app/models/quran/word_font.rb [hunk 1]
```diff
@@ -40,7 +40,11 @@ def transliteration
     word && word.transliteration
   end
 
+  def arabic
+    word && word.token.value
```
Comment (mmahalwy) [inline app/models/quran/word_font.rb]: You'd probably need to preload it here to make sure no queries are happening: https://github.com/quran/quran.com-api/blob/develop/app/controllers/v2/ayahs_controller.rb#L15

**Comment by mmahalwy** [comment]
hmm thats annoying from travis :S

**Comment by mmahalwy** [comment]
salams! If you pull rebase, your tests should pass :)

**Comment by ahmedre** [comment]
merging for now, if the tests still fail, we'll fix insha'Allah on the travis side - @mmahalwy and i are guessing that for some reason, `TRAVIS_PASSWORD` is using the old value for your build.

**Comment by ahmedre** [comment]
(it passed - https://travis-ci.org/quran/quran.com-api/builds/143477259).

**Comment by maryamklabib** [comment]
Yeah I was wondering if it made sense to merge and then run tests, since the problems have been environment-related. It is odd that the password would be different per build, given that the code is run on an external environment unrelated to my setup. But glad it's resolved for now.

**Comment by mmahalwy** [comment]
woot congrats @maryamklabib

**Pull request merged**